src/TTSService.ts

```diff
@@ -100,7 +100,9 @@ export class TTSService {
 
 
 	async play(view: MarkdownView): Promise<void> {
-		let content = view.getViewData();
+		let selectedText = view.editor.getSelection().length > 0 ? view.editor.getSelection() : window.getSelection().toString();
```

Comment:
If user has selected a text in editor mode or in view mode, read that otherwise read the full note.